src/conf.py

```diff
@@ -100,10 +100,10 @@
 #    'sphinx.ext.mathjax'
     'sphinx.ext.imgmath',
     'sphinx.ext.todo',
-    'matplotlib.sphinxext.only_directives',
-    'matplotlib.sphinxext.plot_directive',
-    'IPython.sphinxext.ipython_directive',
-    'IPython.sphinxext.ipython_console_highlighting',
+#    'matplotlib.sphinxext.only_directives',
+#    'matplotlib.sphinxext.plot_directive',
+#    'IPython.sphinxext.ipython_directive',
+#    'IPython.sphinxext.ipython_console_highlighting',
```

Comment:
@francoishamon Do you know why these were added?

Reply:
They were added to enable plotting capabilities: see [here](https://github.com/GEOSX/GEOSX/blob/develop/src/docs/sphinx/tutorials/step08/Tutorial.rst#inspecting-results). In other words, we can include python scripts directly in the documentation.